docs/products/tools/marketplace/guides/wordpress/index.md

```diff
@@ -116,10 +116,10 @@ If you need to reset your admin user's password and you aren't receiving the pas
 
 1. Log in to the Compute Instance using [SSH](/docs/guides/connect-to-server-over-ssh/) or [Lish](/docs/products/compute/compute-instances/guides/lish/).
 
-1.  Navigate to the directory of your WordPress installation:.
+1.  Navigate to the `public_html` directory of your WordPress installation:.
 
     ```command
-    cd /var/www/wordpress
+    cd /var/www/[domain]/public_html
```

Comment:
```suggestion
    cd /var/www/example.com/public_html
```
Please see, https://www.linode.com/docs/guides/linode-writers-formatting-guide/#placeholders

Reply:
Thank you for the PR @timzl